src/Pawn.py

```diff
@@ -56,13 +58,18 @@ def getPossibleMoves(self) -> Iterator[Move]:
             movement = C(0, 2) if self.side == WHITE else C(0, -2)
             advanceTwoPosition = currentPosition + movement
             if self.board.isValidPos(advanceTwoPosition):
-                if self.board.pieceAtPosition(advanceTwoPosition) is None and \
-                   self.board.pieceAtPosition(advanceOnePosition) is None:
+                if (
+                    self.board.pieceAtPosition(advanceTwoPosition) is None
+                    and self.board.pieceAtPosition(advanceOnePosition) is None
```

Comment:
I think we discussed this before. Please indent this correct.

Reply:
```
                if self.board.pieceAtPosition(
                        advanceTwoPosition) is None \
                        and self.board.pieceAtPosition(
                        advanceOnePosition) is None:
```
?